cmd/motion/main.go

```diff
@@ -76,6 +91,30 @@ func main() {
 					return err
 				}
 				store = rbstore
+			} else if cctx.Bool("experimentalSingularityStore") {
+				singularityAPIUrl := cctx.String("remoteSingularityAPIUrl")
+				var client client.Client
+				if singularityAPIUrl != "" {
+					client = httpclient.NewHTTPClient(http.DefaultClient, singularityAPIUrl)
```

Comment:
I left [a question](https://github.com/data-preservation-programs/singularity/pull/94#issuecomment-1653994236) on Singularity repo about the long term vision for this HTTP client. 

If it's not going to be a 1:1 API to the HTTP API then would it make sense to call this something else considering there is now an auto-generated HTTP client library that fully covers the Singularity API?